os/arch/arm/Kconfig

```diff
@@ -104,6 +119,137 @@ config ARCH_CHIP
 	default "amebad"	if ARCH_CHIP_AMEBAD
 	default "amebalite"	if ARCH_CHIP_AMEBALITE
 	default "stm32h745"     if ARCH_CHIP_STM32H745
+	default "imx6"		if ARCH_CHIP_IMX6
+
+config ARCH_HAVE_TRUSTZONE
+        bool
+        default n
+        ---help---
+                Automatically selected to indicate that the ARM CPU supports
+                TrustZone.
+
+choice
+        prompt "TrustZone Configuration"
+        default ARCH_TRUSTZONE_SECURE
+        depends on ARCH_HAVE_TRUSTZONE
+
+config ARCH_TRUSTZONE_SECURE
+        bool "All CPUs operate secure state"
+
+config ARCH_TRUSTZONE_NONSECURE
+        bool "All CPUs operate non-secure state"
+
+config ARCH_TRUSTZONE_BOTH
+        bool "CPUs operate in both secure and non-secure states"
+
+endchoice # TrustZone Configuration
+
+config ARM_THUMB
+        bool "Thumb Mode"
+        default n
+
+config ARM_HAVE_WFE_SEV
+        bool
+        default n
+        ---help---
+                Use WFE and SEV instructions for spinlock to reduce power consumption
+
+config ARM_HAVE_DPFPU32
+        bool
+        select ARCH_HAVE_DPFPU
+        default n
+        ---help---
+                FPU implemented in the VFPv[3|4]-D32 format that supports
+                32 double-precision floating-point registers.
+
+config ARM_HAVE_NEON
+        bool
+        default n
+        select ARM_HAVE_DPFPU32
+        ---help---
+                Decide whether support NEON instruction
+
+config ARM_HAVE_MVE
+        bool
+        default n
+        ---help---
+                Decide whether support MVE instruction
+
+config ARM_FPU_ABI_SOFT
+        bool "Soft Float ABI"
+        default n
+        depends on ARCH_HAVE_FPU
+        ---help---
+                Pass float value via integer register (-mfloat-abi=softfp)
+
+config ARM_DPFPU32
+        bool "FPU with 32 double-precision register"
+        default y
+        depends on ARCH_DPFPU && ARM_HAVE_DPFPU32
+
+config ARM_NEON
+        bool "Advanced SIMD (NEON) Extension"
+        default y
+        depends on ARM_HAVE_NEON && ARM_DPFPU32
+
+config ARM_HAVE_MPU_UNIFIED
+        bool
+        default n
+        ---help---
+                Automatically selected to indicate that the CPU supports a
+                unified MPU for both instruction and data addresses.
+
+config ARM_HAVE_MPCORE
+        bool
+        default n
+        ---help---
+                Decide whether support MPCore extension
+
+config ARM_MPU
```

Comment:
We already have MPU support in CortexM. Why do we need to add these configs in Cortex A porting?